extensions/wikia/TemplateDraft/TemplateConverter.class.php

```diff
@@ -47,6 +53,8 @@ public function convertAsInfobox( $content ) {
 
 		$draft .= "</infobox>\n";
 
+		$draft .= $this->generatePreviewSection( $variables );
```

Comment:
This will break the unit tests for this method. Can you rebase against `CE-templatedraft` so you get the unit tests in this branch?

Alternatively (or additionally), it might be better to separate the method responsibilities for conversion of the infobox text from the preview generation and just call this after `convertAsInfobox` in the controller (will need to account for the variables, but that's a quick change). Would make getting just the infobox conversion reusable rather than always getting the preview text too.